tests/AnnotationTest.php

```diff
@@ -218,6 +218,9 @@ function foo(array $arr = false) : void {}'
      */
     public function providerValidCodeParse(): iterable
     {
+        $codebase = $this->project_analyzer->getCodebase();
```

Comment:
Providers are called *before* `setUp()`, so I don't think you can access properties here.

Reply:
indeed, I just made the fix and pushed that in setUp